usbhid-dump/src/usbhid-dump.c

```diff
@@ -912,7 +912,9 @@ main(int argc, char **argv)
     bool                dump_stream     = false;
     unsigned int        stream_timeout  = 60000;
 
+#ifdef HAVE_SIGACTION
```

Comment:
Ick, again, put all this in a .h file if you really want to do this properly in a way that can be supported.

And I thought windows supported sigaction already.  That feels like a big missing piece if not.